crates/goose-cli/src/commands/configure.rs

```diff
@@ -377,11 +363,11 @@ pub fn configure_extensions_dialog() -> Result<(), Box<dyn Error>> {
             let mut envs = HashMap::new();
             if add_env {
                 loop {
-                    let key = cliclack::input("Environment variable name:")
+                    let key: String = cliclack::input("Environment variable name:")
                         .placeholder("API_KEY")
                         .interact()?;
 
-                    let value = cliclack::password("Environment variable value:")
+                    let value: String = cliclack::password("Environment variable value:")
```

Comment:
server envs are always assumed as a secret which is reasonable (but it could be a config)

![Screenshot 2025-01-23 at 9 51 08 AM](https://github.com/user-attachments/assets/5cb3a2f3-3652-4560-9bf0-a79b2644549f)